provider/fs/src/export/serializers/mod.rs

```diff
@@ -36,7 +36,8 @@ impl From<erased_serde::Error> for Error {
 }
 
 /// A simple serializer trait that works on whole objects.
-pub trait AbstractSerializer: Deref<Target = SyntaxOption> {
+/// TODO: This shouldn't be Deref.
```

Comment:
thought: I assume the plan is to give AbstractSerializer a second method returning the format, yes?